tests/test_client.py

```diff
@@ -156,7 +157,7 @@ async def test_get_access_token() -> None:
             ssl=False,
             allow_redirects=False,
         )
-    await flexmeasures_client.close()
+    # await flexmeasures_client.close()
```

Comment:
I'm not completely sure why it's not needed anymore. If the client is closed it somehow the next test fails because client = None. I think it was probably an uncaught error from that has been resolved by the typing. If you have any ideas I would like to know but otherwise I think we just move forward.